### ISSUE TITLE
Question: About approach in re-training regarding new timeseries data

### ISSUE THREAD

**Issue by rrtjr** [unit8co/darts#919]
Hi team, 

Would it be ok to ask for your thoughts regarding an approach to re-training new data? I have a year's worth of 5-min data and using darts TFT to do a forecast. However, I started noticing some accuracy degradation for this week. Since training the model requires a big chunk of time (3.5 hours per epoch), is there a way where I can leverage the previous training iteration? I am thinking of loading the weights from the previous training and training again this time with the last 2 months up to this week's data. Is this something reasonable to do? Do I need to load just the state dict? Or include optimizers too? I'm just thinking this might be a recipe for overfitting. 

TL;DR: It takes 3 hours per epoch to do a full retrain. Asking for the team's thoughts on how to leverage existing weights to train newer data and lessen training time. 

Thank you nevertheless for your response. I appreciate the flexibility darts has in my research.

**Comment by hrzn** [comment]
Hi, yes, I think it's a very reasonable thing to try, but it's hard to know how well this will work, and I expect it will require some experimentation. All neural-networks based models are conceived such that it is possible to simply call `fit()` on an existing model, which can potentially have already undergone some training before.

**Comment by rrtjr** [comment]
Thank you for this information. This is helpful. I'll try to do some experiments and share if ever there are good findings. Thanks!
